### PR TITLE
Add Cypress tests for microcart

### PR DESCRIPTION
**Pull request by xvyslo05**
## What does this PR do and why?

_Expanding microcart tests to be more bulletproof._

## What is added/fixed/removed?

**Added**
- _Added more test cases, that are covering functionality of microcart quantity elements._

### CODE REVIEW

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/keepProductInCheckoutFromRemovalModal.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default () => cy.get('[data-cy=remove-modal-keep-button]').click()
```
Comment (adamjedlicka) [inline modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/keepProductInCheckoutFromRemovalModal.js:1]: This page object seems to be unused

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromCheckoutFromRemovalModal.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export default () => cy.get('[data-cy=remove-modal-remove-button]').click()
```
Comment (adamjedlicka) [inline modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromCheckoutFromRemovalModal.js:1]: This page object seems to be unused

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromMinicart.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import getProductFromMinicart from '~/cypress/support/pageObjects/minicart/getProductFromMinicart'
```
Comment (adamjedlicka) [inline modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromMinicart.js:1]: This page object seems to be unused

**Review by adamjedlicka — CHANGES_REQUESTED** [review]

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromMinicart.js (hunk 3 above)
Comment (xvyslo05) [reply modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromMinicart.js:1]: not used in test files, but used in page objects 
image

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/keepProductInCheckoutFromRemovalModal.js (hunk 1 above)
Comment (xvyslo05) [reply modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/keepProductInCheckoutFromRemovalModal.js:1]: yup, created for the next PR already. Altogether with the `data-cy` attribute in the source file. I decided to keep it here, even though it is not used yet.

File: modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromCheckoutFromRemovalModal.js (hunk 2 above)
Comment (xvyslo05) [reply modules/theme-tailwind-magento/cypress/support/pageObjects/minicart/removeProductFromCheckoutFromRemovalModal.js:1]: yup, created for the next PR already. Altogether with the `data-cy` attribute in the source file. I decided to keep it here, even though it is not used yet.

**Review by adamjedlicka — APPROVED** [review]

**Pull request merged**